src/components/Wallet/WalletInfoCard.jsx

```diff
@@ -93,7 +96,6 @@ const WalletInfoCard = ({
             </>
           )}
         </Grid>
-        {children}
```

Comment:
Shoot, sorry. I shouldn't have approved/merged this without discussing further. This is here to allow for this component to be more flexible/useful in other situations. Having children allows a developer that is using this wallet info card to insert other information that might be helpful in the case of their particular info card. The idea was to make this reusable so that a whole new info card wouldn't need to be re-written in case something like the current date, progress bar, etc. was needed further down the line.

Reply:
> Shoot, sorry. I shouldn't have approved/merged this without discussing further. This is here to allow for this component to be more flexible/useful in other situations. Having children allows a developer that is using this wallet info card to insert other information that might be helpful in the case of their particular info card. The idea was to make this reusable so that a whole new info card wouldn't need to be re-written in case something like the current date, progress bar, etc. was needed further down the line.

That makes sense but maybe we can add that in the future if it's ever needed since it's not used anywhere?  I removed it because it seemed like either a premature optimization or unintentional.

Reply:
The issue with "since it's not used anywhere" is that we won't know if it's being used if it's in a component library. Having this is kind of like defining its API and the intention that it could be used this way. Obviously it's not in a component library yet, but ideally it will be soon and I was hoping to have the API set for the components that are ready to be migrated.

Reply:
Feel free to add it back if you want in the cleanup PR or we can add it back when we move it to the component library.  When that happens I think we will want to have more formal design reviews of components anyway in order to determine if we want this to be able to be extended, etc.